arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/raise/Trace.kt

```diff
@@ -0,0 +1,42 @@
+@file:JvmMultifileClass
+@file:JvmName("RaiseKt")
+package arrow.core.raise
+
+import kotlin.coroutines.cancellation.CancellationException
+import kotlin.jvm.JvmMultifileClass
+import kotlin.jvm.JvmName
+
+@RequiresOptIn("This API is experimental, and may change in the future.")
+public annotation class ExperimentalTraceApi
+
+/** Tracing result of `R`. Allows to inspect `R`, and the traces from where it was raised. */
```

Comment:
```suggestion
/** Tracing result. Allows to inspect the traces from where raise was called. */
```
Since Trace doesn't include the raised value anymore, the kdoc should reflect that change